CONTRIBUTING.md

```diff
@@ -34,3 +92,76 @@ To run all tests:
 $ cd tests
 $ ./test.sh -a
 ```
+
+## Releasing
+
+From a clean instance of main, perform the following actions to release a new version
+of this plugin:
+
+- Update the version number in [`galaxy.yml`](galaxy.yml) and [`CHANGELOG.md`](CHANGELOG.md)
+    - Verify that all changes for this version in `CHANGELOG.md` are clear and accurate,
+      and are followed by a link to their respective issue
+    - Create a PR with these changes
+
+- Create an annotated tag with the new version, formatted as `v##.##.##`
+    - This will kick off an automated script which publish the release to
+      [Ansible Galaxy](https://galaxy.ansible.com/cyberark/conjur)
+
+- Create the release on GitHub for that tag
+    - Build the release package with `./ci/build_release`
+    - Attach package to Github Release
+
+## Set up Conjur Open Source and Ansible control node
+  - ### Setup a conjur OSS Environment
+
+    -   Pull the Docker image
+    -   Generate the master key
+    -   Load master key as an environment variable
+    -   Start the Conjur OSS environment
+    -   Create admin account
+    -   Connect the Conjur client to the Conjur server
+
+  - ### Granting a Conjur identity to Ansible hosts
+    To grant your Ansible host a Conjur identity, you first must install the Conjur Ansible Role in your playbook
+          directly
+      ```sh-session
+      ansible-galaxy install cyberark.conjur-host-identity
+      ```
+      Once you've done this, you can configure each Ansible node with a Conjur identity by including a section like the example below in your Ansible playbook:
+
+      ```sh-session
+      - hosts: servers
+        roles:
+          - role: cyberark.conjur-host-identity
+            conjur_appliance_url: 'https://conjur.myorg.com',
+            conjur_account: 'myorg',
+            conjur_host_factory_token: "{{lookup('env', 'HFTOKEN')}}",
+            conjur_host_name: "{{inventory_hostname}}"
+      ```
+      First we register the host with Conjur, adding it into the layer specific to the provided host factory token, and then installs Summon with the Summon Conjur provider for secret retrieval from Conjur.
+
+## Load policy to set up Conjur Ansible integration
+
+
+  Policy defines Conjur entities and the relationships between them.  An entity can be a policy, a host, a user, a layer, a group, or a variable.
+
+  ```sh
+    docker exec conjur_client conjur policy load root /policy/root.yml
+  ```
+## Create Ansible managed nodes
+
+  Set all required variables, ensure `Conjur_host_factory_token` is set . Install `ca-certificates` and place Conjur public SSL certificate. Symlink Conjur public SSL certificate into /etc/ssl/certs .
+
+## Use conjur host identity to set up Conjur identity on managed nodes
```

Comment:
The content of this section should be replaced with the content of the section `### Granting a Conjur identity to Ansible hosts`. This section currently tells the user _how_ the role will configure their managed nodes - we should focus on usage instructions here.

Reply:
Done